README.md

```diff
@@ -2,13 +2,84 @@
 
 [![CircleCI](https://circleci.com/gh/ghost-inspector/node-ghost-inspector/tree/stable.svg?style=svg)](https://circleci.com/gh/ghost-inspector/node-ghost-inspector/tree/stable)
 
-The official Node.js package for interacting with [Ghost Inspector's API](https://ghostinspector.com/docs/api/).
+The official Node.js package and CLI for interacting with [Ghost Inspector's API](https://ghostinspector.com/docs/api/).
 
-## Installing with [npm](https://www.npmjs.com/)
+## Jump to...
 
-`npm install ghost-inspector`
+ * [Installation](#installation)
+ * [Quick Start](#quick-start)
+ * [CLI Usage](#cli-usage)
+ * [Node.js Client Usage](#nodejs-client-usage)
+ * [Contributing](#contributing)
 
-## Usage
+## Installation
+
+Our official Node.js package is available from [npm](https://www.npmjs.com/), you can install it with the following command:
+
+```
+npm install ghost-inspector
+```
+
+In order to use the CLI, install the package globally:
+
+```
+npm install -g ghost-inspector
+```
+
+## Quick Start
+
+```js
+const GhostInspector = require('ghost-inspector')('[api-key]')
+
+// execute a test
+try {
+  const [results, passing, screenshotPassing] = await GhostInspector.executeTest(
+    '[test-id]',
+    options,
+  )
+  console.log('Passing: ', passing)
+} catch (err) {
+  console.error(err)
+}
+
+```
+
+## CLI Usage
+
+View all available commands for the CLI:
+```
+❯ ghost-inspector --help
+ghost-inspector <command>
+
+Commands:
+  ghost-inspector folder <command>        Manage folders within your Ghost Inspector account.
+  ghost-inspector organization <command>  Access organization details.
+  ghost-inspector suite-result <command>  Manage suite results within your Ghost Inspector account.
+  ghost-inspector suite <command>         Manage suites within your Ghost Inspector account.
+  ghost-inspector test-result <command>   Manage test results within your Ghost Inspector account.
+  ghost-inspector test-runner-ips         Fetch a list of test runner IP addresses by region.
+  ghost-inspector test <command>          Manage tests within your Ghost Inspector account.
+
+Options:
+  --version  Show version number                                                           [boolean]
+  --apiKey   Your Ghost Inspector API key.                                                [required]
+  --help     Show help                                                                     [boolean]
+```
+
+You can also view usage help for each sub-command:
+
+```
+❯ ghost-inspector test --help
+...
+
+❯ ghost-inspector test execute --help
+...
+```
```

Comment:
I think it's worth adding one more block here with a couple of real examples, instead of just the `--help` usage.  Things like this:

```
> ghost-inspector test execute xxx  --apiKey xxx
```